datahub-classify/setup.py

```diff
@@ -27,9 +29,12 @@ def get_long_description():
     "flake8>=3.8.3",
     "flake8-tidy-imports>=4.3.0",
     "isort>=5.7.0",
-    "mypy>=0.901,<0.920",
+    # Because of https://github.com/python/mypy/issues/13627 issue, set the mypy version to 0.981, we can revisit this in future
+    "mypy>==0.981",
     "pytest-cov>=2.8.1",
-    "scikit-learn",
+    "scikit-learn==1.1.2",
+    "pandas>=1.5.0,<=1.5.1",
```

Comment:
why is it now possible to move the pandas dep from base_requirements to dev_requirements?

Reply:
We removed the dependency of pandas from the production code. Now, it is just required in the testing code.

Reply:
that's great!